cmd/minikube/cmd/config/validations.go

```diff
@@ -75,3 +76,11 @@ func IsValidPath(name string, path string) error {
 	}
 	return nil
 }
+
+func IsValidUrl(string, isoURL string) error {
```

Comment:
> I meant if we use "file://" in mac or in Linux that didn't work to get file downloaded using file URI.

I am pretty sure this works

> so as per parsing concern file:/, file:// and file:/// all are valid from ParseURI method.

I think this is not quite correct implementation wise, but if it is like this we don't have to touch this right now.